src/coreclr/vm/exceptionhandling.cpp

```diff
@@ -572,7 +585,7 @@ static void PopExplicitFrames(Thread *pThread, void *targetSp, void *targetCalle
     }
 }
 
-EXTERN_C EXCEPTION_DISPOSITION
+EXTERN_C EXCEPTION_DISPOSITION __cdecl
```

Comment:
Are these handlers ever passed to some Windows API that requires cdecl calling convention?

If not, we should not need `__cdecl` on them since we control all callers.

Reply:
Ok, I see it in the asm code now.